packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/JoinInButton/JoinInButton.js

```diff
@@ -1,24 +1,27 @@
 import { Button } from 'grommet'
-import Link from 'next/link'
-import { object } from 'prop-types'
 import styled from 'styled-components'
 import { useTranslation } from 'next-i18next'
+import { useRouter } from 'next/router'
+
+import addQueryParams from '@helpers/addQueryParams'
```

Comment:
I consolidated JoinInButtonContainer into this one file. It's a simple button component, no need for connectors and containers.